api.raml

```diff
@@ -352,6 +361,22 @@ schemas:
           description: |
             The corresponding user record (matching on the `employee_id` URL
             parameter) was updated to match the given data.
+          body:
+            application/json:
+              schema: User
+              example: |
+                {
+                  "employee_id": "12345",
+                  "first_name": "John",
+                  "last_name": "Smith",
+                  "display_name": "John Smith",
+                  "username": "john_smith",
+                  "email": "john_smith@example.com",
+                  "active": "yes",
```

Comment:
@wcjr Since the example call to `PUT` has `"active": "no"`, what would you think of making `active` equal to `"no"` here in the response example? By no means necessary, it just seems like a nice nuance.